src/Akka.Hosting.Tests/Akka.Hosting.Tests.csproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Akka.Hosting.TestKit\Akka.Hosting.TestKit.csproj" />
```

Comment:
Needed to add the Hosting.TestKit in order to test the `BackgroundService` issue with #208